ch03.adoc

```diff
@@ -139,7 +139,7 @@ Use of these attributes for data packing, which is their most important applicat
 [[long-name, Section 3.2, "Long Name"]]
 === Long Name
 
-The **`long_name`** attribute is defined by the NUG to contain a long descriptive name which may, for example, be used for labeling plots.
+The **`long_name`** attribute is defined by the NUG <<NUG>> to contain a long descriptive name which may, for example, be used for labeling plots.
```

Comment:
could this be just <<NUG>>?

Reply:
Fixed: 3df131ec211a578713034ae3f65518fe202e1a96